libs/sdk-ui-dashboard/src/presentation/drill/DrillConfigPanel/DrillToUrl/AttributeUrlSection.tsx

```diff
@@ -51,23 +51,17 @@ export const AttributeUrlSection: React.FC<AttributeUrlSectionProps> = (props) =
                         if (!isDrillToAttributeUrl(item.drillDefinition)) {
                             return null;
                         }
+                        const target = item.drillDefinition.target;
                         return (
                             <AttributeUrlSectionItem
-                                key={objRefToString(item.drillDefinition.target.displayForm)}
+                                key={objRefToString(target.displayForm)}
                                 item={item.drillDefinition}
                                 isSelected={areObjRefsEqual(
-                                    item.drillDefinition.target.hyperlinkDisplayForm,
+                                    target.hyperlinkDisplayForm,
                                     selected || undefined,
                                 )}
-                                onClickHandler={
-                                    () => {
-                                        /**/
-                                    }
-                                    // onClickHandler(
-                                    //     e,
-                                    //     item.ref,
-                                    //     item.ref, // TODO
-                                    // )
+                                onClickHandler={(e) =>
+                                    onClickHandler(e, target.displayForm, target.hyperlinkDisplayForm)
```

Comment:
Remove lamda here and decompose  each attribute in onClickHandler = useCallback( ...  than you will have memoized callback ;-)